opentelemetry-http/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- Add `reqwest-rustls-webkpi-roots` feature flag to configure `reqwest` to use embedded `webkpi-roots`.
```

Comment:
could you link the reqwest page for this feature?

Reply:
done